lib/src/blocs/signup_bloc/signup_event.dart

```diff
@@ -0,0 +1,27 @@
+part of 'signup_bloc.dart';
+
+abstract class SignupEvent extends Equatable {
+  final String fullName;
+  final String email;
+  final String password;
```

Comment:
should we need these properties somewhere for this abstract?

Reply:
This is the Signup file, I don't use it in my screen, when we merge code, this file will change, I just include the name screen for logic testing. Now I have deleted it